src/implementations/twig/components/table/table.story.js

```diff
@@ -28,7 +40,15 @@ const getArgTypes = () => ({
   },
 });
 
-const prepareData = (data, args) => Object.assign(data, args);
+const prepareData = (data, args) => {
```

Comment:
this way we break the simple control, we should still return an Object.assign to get it working again.